packages/block-library/src/query/edit/query-inspector-controls.js

```diff
@@ -13,6 +13,7 @@ import {
 	TextControl,
 	FormTokenField,
 	SelectControl,
+	CheckboxControl,
```

Comment:
This will be subject to designer but maybe `ToggleControl` feels better?

Reply:
Was about to suggest the same thing. The toggle control I often compare to a light-switch — the effect is instant. Whereas a checkbox, not always but often, is combined with a "Save" or "Apply" button somewhere on the page.

Reply:
Good point. Switched it to a `ToggleControl` :+1: